plugins/@grouparoo/rollbar/package-lock.json

```diff
@@ -0,0 +1,17982 @@
+{
```

Comment:
plugins/@grouparoo/rollbar/package-lock.json should not exist - this should be handled by pnpm